charts/preview/templates/operator-role.yaml

```diff
@@ -32,6 +32,12 @@ rules:
       - get
       - delete
       - update
+  - apiGroups:
```

Comment:
Duplicated? See line 41

Reply:
Just the same apiGroup that is being used for different rules. The verbs get allocated to all the resources so this duplicate is the only way to associated different verbs to differents resources.